### ISSUE TITLE
Implement SQS-based communication

### ISSUE THREAD

**Issue by mcopik** [spcl/fmi#1]
One communication channel we have yet to cover so far is AWS SQS. We could use it to communicate between functions, with the main limitation being the low message size (256 kB) and the need for base64 encoding.

Steps for the solution:
* [ ] Allocate the SQS client and select a proper queue.
* [ ] Implement a new indirect channel.
* [ ] Implement splitting serialized data across multiple packets.
* [ ] Add tests covering the new communication method.

**Comment by whiz-Tuhin** [comment]
Hi @mcopik,
I am going through the FMI paper and your thesis and am interested in contributing to this project.
Should I get started on this issue ?

**Comment by mcopik** [comment]
@whiz-Tuhin yes, please :-) Implementing SQS using existing interfaces should be straightforward. 

Let me know if you have any questions!
